packages/contracts-rfq/contracts/FastBridgeV2.sol

```diff
@@ -79,7 +79,12 @@ contract FastBridgeV2 is Admin, IFastBridgeV2 {
         // check bridge params
         if (params.dstChainId == block.chainid) revert ChainIncorrect();
         if (params.originAmount == 0 || params.destAmount == 0) revert AmountIncorrect();
-        if (params.originToken == address(0) || params.destToken == address(0)) revert ZeroAddress();
+        if (
+            params.originToken == address(0) || 
+            params.destToken == address(0) || 
+            params.sender == address(0) || 
+            params.to == address(0)
+        ) revert ZeroAddress();
```

Comment:
Does two independent checks make more sense to you here? If not, this needs a `// forgefmt: disable-next-item` before this statement to avoid being formatted in a very ugly way (try doing `forge fmt` and the result is much less readable than what it is now).

Reply:
makes sense to me for readability with forge fmt in mind, but not logical sense :)

i went ahead and made it 2 lines. Also confirmed via forge tests that it only increases gas by a couple units

Reply:
lgtm